### PR TITLE
Support markdown #8

### PR DESCRIPTION
**Pull request by kasunben**
Fix for #8

### CODE REVIEW

File: src/manager/components/CommentForm/index.js [hunk 1]
```diff
@@ -1,7 +1,24 @@
 import React, { Component } from 'react';
 import Textarea from 'react-textarea-autosize';
+import marked from 'marked';
 import style from './style';
```
Comment (arunoda) [inline src/manager/components/CommentForm/index.js:4]: We can't import CSS like this.
Check this: https://github.com/kadirahq/storybook-addon-knobs/pull/26/files#diff-50f961defa6d9bfc44bba2257fc2ff7aR3

File: src/manager/components/CommentList/style.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+.comment-content p {
```
Comment (arunoda) [inline src/manager/components/CommentList/style.css]: Could you try different Markdown elements and see they looks great here.
Try to match the style to look like GitHub if needed.

**Review by arunoda — CHANGES_REQUESTED** [review]

File: src/manager/components/CommentForm/index.js [hunk 3]
```diff
@@ -44,6 +44,8 @@ export default class CommentForm extends Component {
       return;
     }
 
+    console.log(marked(text))
```
Comment (arunoda) [inline src/manager/components/CommentForm/index.js]: Remove this.

File: src/manager/components/CommentList/index.js [hunk 4]
```diff
@@ -1,8 +1,62 @@
 import React, { Component } from 'react';
 import moment from 'moment';
 import renderHTML from 'react-render-html';
+import insertCss from 'insert-css';
 import style from './style';
-import './style.css';
+
+const customStyle = `
+  .comment-content p {
```
Comment (arunoda) [inline src/manager/components/CommentList/index.js:8]: Where did you get this style?
Is this custom one or?

**Review by arunoda — CHANGES_REQUESTED** [review]

File: src/manager/components/CommentList/index.js (hunk 4 above)
Comment (kasunben) [reply src/manager/components/CommentList/index.js:8]: Yes!, custom styles.
follows this https://jbt.github.io/markdown-editor/

Comment (arunoda) [reply src/manager/components/CommentList/index.js:8]: Okay. kwl.

**Review by arunoda — APPROVED** [review]

**Pull request merged**